# HG changeset patch
# User Rik <rik@octave.org>
# Date 1390593484 28800
#      Fri Jan 24 11:58:04 2014 -0800
# Branch stable
# Node ID 7655206949925ad5a98f01be6e85088097da1325
# Parent  ac664a250eab350b98961709855c142ab4d169ae
doc: Add dblist to list of commands from 'help debug'.

* debug.m: Add dblist to list of commands.

diff --git a/scripts/miscellaneous/debug.m b/scripts/miscellaneous/debug.m
--- a/scripts/miscellaneous/debug.m
+++ b/scripts/miscellaneous/debug.m
@@ -32,19 +32,23 @@
 ##
 ## @item dbstatus
 ## List all breakpoints.
 ##
 ## @item dbwhere
 ## Report the current file and line number where execution is stopped.
 ##
 ## @item dbtype
-## List the function where execution is currently stopped, enumerating
+## Display the code of the function being debugged, enumerating
 ## the line numbers.
 ##
+## @item dblist
+## List 10 lines of code centered around the line number where execution is
+## stopped.
+##
 ## @item  dbstep
 ## @itemx dbnext
 ## Execute (step) one or more lines, follow execution into (step into) a
 ## function call, or execute until the end of a function (step out), and
 ## re-enter debug mode.
 ##
 ## @item dbcont
 ## Continue normal code execution from the debug prompt.
